app/src/main/java/com/nervousfish/nervousfish/modules/pairing/ZXingQRAdapter.java

```diff
@@ -0,0 +1,29 @@
+package com.nervousfish.nervousfish.modules.pairing;
+
+import com.nervousfish.nervousfish.service_locator.IServiceLocator;
+import com.nervousfish.nervousfish.service_locator.IServiceLocatorCreator;
+import com.nervousfish.nervousfish.service_locator.ModuleWrapper;
+
+/**
+ * An adapter to the ZXing QR code library
+ */
+public final class ZXingQRAdapter implements IQRHandler {
```

Comment:
I think ZXing QR code is to specific for now, we know we want to support QR codes but haven't discussed what QR codes yet, so I agree with @clenrock on this one 🙂